lib/management/sso.ts

```diff
@@ -121,7 +128,27 @@ const withSSOSettings = (sdk: CoreSdk, managementKey?: string) => ({
         queryParams: { tenantId },
         token: managementKey,
       }),
-      (data) => data,
+      (data) => {
+        const readySettings = data as any;
+        if (readySettings.oidc) {
+          readySettings.oidc = {
+            ...readySettings.oidc,
+            attributeMapping: readySettings.oidc.userAttrMapping,
+          };
+          delete readySettings.oidc.userAttrMapping;
+        }
+        const mappings: RoleMappings = [];
+        readySettings.saml?.groupsMapping?.forEach((gm: any) => {
+          const rm = gm;
+          rm.roleName = rm.role.name;
+          delete rm.role;
+          mappings.push(rm);
+        });
+        if (readySettings.saml?.groupsMapping) {
+          readySettings.saml.groupsMapping = mappings;
+        }
+        return data;
```

Comment:
pointers and all, but shouldn't we return `readySettings` here?

Reply:
done